src/ontology/cl-edit.owl

```diff
@@ -9683,7 +9695,7 @@ SubClassOf(obo:CL_0000750 obo:CL_0000748)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:14689473") obo:IAO_0000115 obo:CL_0000751 "A bipolar neuron found in the retina that is synapsed by rod photoreceptor cells but not by cone photoreceptor cells.  These neurons depolarize in response to light.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000751 "FMA:67750")
 AnnotationAssertion(rdfs:label obo:CL_0000751 "rod bipolar cell")
-SubClassOf(obo:CL_0000751 obo:CL_0000748)
+SubClassOf(obo:CL_0000751 obo:CL_0000749)
```

Comment:
OK, but of note potential exceptions in fish:

"The mammalian rod bipolar cell is an ON-type cell (123) utilizing metabotropic APB-sensitive, glutamate receptors (31, 32, 64, 124). In fish, the comparable cell type is the Mb bipolar cell (125). In fish retinas, rod-contacting bipolar cells also contact cones. **Five such types are found in goldfish retina, including multiple ON types with axon terminals branching in sublamina b, and likely OFF types with axon terminals branching in sublamina a (14).** "
https://webvision.med.utah.edu/book/part-v-phototransduction-in-rods-and-cones/bipolar-cell-pathways-in-the-vertebrate-retina/

and 
https://doi.org/10.1016/0014-4835(79)90128-3

Reply:
CL:0000748 'rod bipolar cell' is not restricted to mammals. 

To justify the change from
`SubClassOf(obo:CL_0000751 obo:CL_0000748)`
to
`SubClassOf(obo:CL_0000751 obo:CL_0000749)`
in this PR, I believe the restriction needs to be added. Adding the restriction to mammals would also align with the second sentence of the text def: "These neurons depolarize in response to light."

Another reference:

"Bipolar cell responses can be either depolarizing (ON) or hyperpolarizing (OFF); in lower vertebrates, bipolar cells receive mixed rod and cone input, whereas in mammals, input is mostly segregated into 14 classes of cone ON and OFF cells and a single rod ON bipolar cell. **We show that lamprey, like mammals, have rod bipolar cells with little or no cone input, but these cells are OFF rather than ON.**
...
 In the process of investigating rod and cone input to these cells, we discovered an abundant cell type with cell bodies abutting the photoreceptor synaptic layer and axons penetrating deep into the IPL, **reminiscent of the mammalian rod bipolar cell** (Dacheux and Raviola, 1986). These cells receive input almost exclusively from rods but are OFF instead of ON, suggesting that the evolution of bipolar cells in vertebrate retina may have been more fluid than previously thought."
https://doi.org/10.1242/jeb.243949

Reply:
Agree, I have added `'only in taxon' some Mammalia`